tutorials/notebooks/mct_features_notebooks/keras/example_keras_qat.ipynb

```diff
@@ -193,9 +193,9 @@
     "from model_compression_toolkit import DefaultDict\n",
     "from model_compression_toolkit.constants import FLOAT_BITWIDTH\n",
     "from model_compression_toolkit.target_platform_capabilities.constants import KERNEL_ATTR, KERAS_KERNEL, BIAS_ATTR, BIAS\n",
-    "\n",
-    "tp = mct.target_platform\n",
-    "\n",
+    "from mct_quantizers import QuantizationMethod\n",
+    "from model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema import schema, TargetPlatformCapabilities, Signedness, \\\n",
```

Comment:
you already have an import from this path above.
also, to make it easier on the eye, I would import:
`import model_compression_toolkit.target_platform_capabilities.schema.mct_current_schema as schema`
and then use `schema.<class_name>` for all schema classes.
same for the running tutorial